.ci/scripts/create_simulator.sh

```diff
@@ -11,5 +11,19 @@ readonly DEVICE_NAME=Flutter-iPhone
 readonly DEVICE=com.apple.CoreSimulator.SimDeviceType.iPhone-14
 readonly OS=com.apple.CoreSimulator.SimRuntime.iOS-16-4
 
+# Delete any existing devices named Flutter-iPhone. Having more than one may
+# cause issues when builds target the device.
```

Comment:
For reference, here's an example of this problem in action: https://ci.chromium.org/ui/p/flutter/builders/try/Mac_arm64%20custom_package_tests%20master/7040/overview

Unclear if it's a problem new to macOS 13